Microsoft.Alm.Authentication/Src/Global.cs

```diff
@@ -67,6 +63,22 @@ public static string UserAgent
 
         }
 
+        public static int RequestTimeout
+        {
+            get
+            {
+                lock (_syncpoint)
```

Comment:
So why the scoped format for the `get` and the inline format for the `set`?

Perhaps just make both inline?

```C#
get { lock (_syncpoint) return _requestTimeout; }
set { lock (_syncpoint) _requestTimeout = value; }
```